### PR TITLE
[BULK] DocuTune - Updates to Azure AD rebranding guidance and DocuTune configuration (part 6)

### PR DESCRIPTION
**Pull request by alexbuckgit**
Applying Azure AD rebranding changes as announced on July 11, 2023 and outlined in https://aka.ms/AzureADNewName. Changes are part of the Microsoft-wide rebranding effort.

CorrelationId: 2520d58e-25ff-469e-af14-2834d2a1505e
PointOfContact: @CelesteDG

#docutune

### CODE REVIEW

**Comment by alexbuckgit** [comment]
#docutune-draft

**Review by mikefrobbins — DISMISSED** [review]
LGTM

**Comment by CelesteDG** [comment]
@mikefrobbins @alexbuckgit Has this build stalled? Do we need to close and reopen the PR?

File: src/Storage/Storage.Management/help/New-AzStorageAccount.md [hunk 1]
```diff
@@ -120,12 +120,14 @@ New-AzStorageAccount -ResourceGroupName "MyResourceGroup" -AccountName "mystorag
 
 This command creates a Storage account with Hierarchical Namespace enabled, Sftp enabled, and localuser enabled.
 
-### Example 6: Create a Storage account with Azure Files AAD DS Authentication, and enable large file share.
+<a name='example-6-create-a-storage-account-with-azure-files-aad-ds-authentication-and-enable-large-file-share'></a>
+
```
Comment (BethanyZhou) [inline src/Storage/Storage.Management/help/New-AzStorageAccount.md:124]:
```suggestion
```

**Review by BethanyZhou — CHANGES_REQUESTED** [review]

File: src/Storage/Storage.Management/help/Set-AzStorageAccount.md [hunk 2]
```diff
@@ -172,7 +172,9 @@ Set-AzStorageAccount -ResourceGroupName "MyResourceGroup" -Name "mystorageaccoun
 
 The command upgrade a Storage account with Kind "Storage" or "BlobStorage" to "StorageV2" kind Storage account.
 
-### Example 10: Update a Storage account by enable Azure Files AAD DS Authentication and set DefaultSharePermission.
+<a name='example-10-update-a-storage-account-by-enable-azure-files-aad-ds-authentication-and-set-defaultsharepermission'></a>
+
```
Comment (BethanyZhou) [inline src/Storage/Storage.Management/help/Set-AzStorageAccount.md:176]:
```suggestion
```

File: src/Ssh/Ssh/help/Export-AzSshConfig.md [hunk 3]
```diff
@@ -39,20 +39,22 @@ Export-AzSshConfig -ResourceId <String> -ConfigFilePath <String> [-PublicKeyFile
...
 ```
 
-When a -LocalUser is not supplied, the cmdlet will attempt to create a certificate to login using Azure AD. This is currently only supported for resources running Linux OS.
-When using Azure AD to login to resource, the Host name in the configuration entry will be "{resource group name}-{resource name}", or "{ip address}" for Azure VMs.
+When a -LocalUser is not supplied, the cmdlet will attempt to create a certificate to login using Microsoft Entra ID. This is currently only supported for resources running Linux OS.
+When using Microsoft Entra ID to login to resource, the Host name in the configuration entry will be "{resource group name}-{resource name}", or "{ip address}" for Azure VMs.
 
-### Example 2: Export a SSH configuration file for connecting to the Public Ip of an Azure Virtual Machine using AAD issued certificates.
+<a name='example-2-export-a-ssh-configuration-file-for-connecting-to-the-public-ip-of-an-azure-virtual-machine-using-aad-issued-certificates'></a>
+
```
Comment (BethanyZhou) [inline src/Ssh/Ssh/help/Export-AzSshConfig.md:56]:
```suggestion
```

**Review by mikefrobbins — APPROVED** [review]
LGTM

**Comment by BethanyZhou** [comment]
FPs are reported. please help force-merge
```
 "Module","Cmdlet","Example","Line","RuleName","ProblemId","Severity","Description","Extent","Remediation"
  "Az.Ssh","Enter-AzVM","1","1","Invalid_Cmdlet","5000","1","Enter-AzVM is not a valid command name.","Enter-AzVM -ResourceGroupName myRg -Name myMachine","Check the spell of Enter-AzVM."
  "Az.Ssh","Enter-AzVM","2","1","Invalid_Cmdlet","5000","1","Enter-AzVM is not a valid command name.","Enter-AzVM -ResourceGroupName myRg -Name myMachine -LocalUser azureuser -PrivateKeyFile ./id_rsa -CertificateFile ./cert","Check the spell of Enter-AzVM."
  "Az.Ssh","Enter-AzVM","3","1","Invalid_Cmdlet","5000","1","Enter-AzVM is not a valid command name.","Enter-AzVM -ResourceGroupName myRg -Name myMachine -LocalUser azureuser -PrivateKeyFile ./id_rsa","Check the spell of Enter-AzVM."
  "Az.Ssh","Enter-AzVM","4","1","Invalid_Cmdlet","5000","1","Enter-AzVM is not a valid command name.","Enter-AzVM -ResourceGroupName myRg -Name myMachine -LocalUser azureuser","Check the spell of Enter-AzVM."
  "Az.Ssh","Enter-AzVM","5","1","Invalid_Cmdlet","5000","1","Enter-AzVM is not a valid command name.","Enter-AzVM -Ip 1.2.3.4","Check the spell of Enter-AzVM."
  "Az.Ssh","Enter-AzVM","6","1","Invalid_Cmdlet","5000","1","Enter-AzVM is not a valid command name.","Enter-AzVM -ResourceGroupName myRg -Name myMachine -ResourceType Microsoft.HybridCompute/machines","Check the spell of Enter-AzVM."
  "Az.Ssh","Enter-AzVM","7","1","Invalid_Cmdlet","5000","1","Enter-AzVM is not a valid command name.","Enter-AzVM -ResourceGroupName myRg -Name myMachine -PrivateKeyFile ./id_rsa -PublicKeyFile ./id_rsa.pub","Check the spell of Enter-AzVM."
  "Az.Ssh","Enter-AzVM","8","1","Invalid_Cmdlet","5000","1","Enter-AzVM is not a valid command name.","Enter-AzVM -ResourceGroupName myRg -Name myMachine -LocalUser username -Rdp","Check the spell of Enter-AzVM."
  "Az.Ssh","Export-AzSshConfig","1","1","Invalid_Cmdlet","5000","1","Export-AzSshConfig is not a valid command name.","Export-AzSshConfig -ResourceGroupName myRg -Name myMachine -ConfigFilePath ./sshconfig.config","Check the spell of Export-AzSshConfig."
  "Az.Ssh","Export-AzSshConfig","2","1","Invalid_Cmdlet","5000","1","Export-AzSshConfig is not a valid command name.","Export-AzSshConfig -Ip 1.2.3.4 -ConfigFilePath ./sshconfig.config","Check the spell of Export-AzSshConfig."
  "Az.Ssh","Export-AzSshConfig","3","1","Invalid_Cmdlet","5000","1","Export-AzSshConfig is not a valid command name.","Export-AzSshConfig -ResourceGroupName myRg -Name myVm -LocalUser azureuser -CertificateFile ./cert -PrivateKeyFile ./id_rsa -ConfigFilePath ./sshconfig.config","Check the spell of Export-AzSshConfig."
  "Az.Ssh","Export-AzSshConfig","4","1","Invalid_Cmdlet","5000","1","Export-AzSshConfig is not a valid command name.","Export-AzSshConfig -ResourceGroupName myRg -Name myVm -LocalUser azureuser -PrivateKeyFile ./id_rsa -ConfigFilePath ./sshconfig.config","Check the spell of Export-AzSshConfig."
  "Az.Ssh","Export-AzSshConfig","5","1","Invalid_Cmdlet","5000","1","Export-AzSshConfig is not a valid command name.","Export-AzSshConfig -ResourceGroupName myRg -Name myVm -LocalUser azureuser -ConfigFilePath ./sshconfig.config","Check the spell of Export-AzSshConfig."
  "Az.Ssh","Export-AzSshConfig","6","1","Invalid_Cmdlet","5000","1","Export-AzSshConfig is not a valid command name.","Export-AzSshConfig -ResourceGroupName myRg -Name myVm -KeysDestinationFolder /home/user/mykeys -ConfigFilePath ./sshconfig.config","Check the spell of Export-AzSshConfig."
  "Az.Ssh","Export-AzSshConfig","7","1","Invalid_Cmdlet","5000","1","Export-AzSshConfig is not a valid command name.","Export-AzSshConfig -Ip * -ConfigFilePath ./sshconfig.config","Check the spell of Export-AzSshConfig."
  "Az.Ssh","Export-AzSshConfig","8","1","Invalid_Cmdlet","5000","1","Export-AzSshConfig is not a valid command name.","Export-AzSshConfig -ResourceGroupName myRg -Name myVm -ResourceType Microsoft.Compute/virtualMachines -ConfigFilePath ./sshconfig.config","Check the spell of Export-AzSshConfig."
  "Az.TimeSeriesInsights","New-AzTimeSeriesInsightsAccessPolicy","1","1","Invalid_Cmdlet","5000","1","New-AzTimeSeriesInsightsAccessPolicy is not a valid command name.","New-AzTimeSeriesInsightsAccessPolicy -EnvironmentName tsitest001 -ResourceGroupName testgroup -PrincipalObjectId ce74a389-b5e8-4f16-89c7-787031ddd903 -Role Contributor -Name policy001","Check the spell of New-AzTimeSeriesInsightsAccessPolicy."

```

**Review by BethanyZhou — APPROVED** [review]

**Pull request merged**